js-core/homeworks/homework-15/src/main.js

```diff
@@ -0,0 +1,386 @@
+Выглядеть должно так же: https://aleksandra-maslennikova.github.io/telephone-book/index.html
+*/
+
+const contentForThead = ['Name', 'Last Name', 'Email'];
+
+const users = [
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    
+    
+];
+
+const footerTabs = [
+    'Contacts',
+    'Keypad',
+    'Edit contact',
+    'User',
+    'Add user'
+];
+
+const idOfIcons = [
+    'search',
+    'th',
+    'pencil',
+    'user',
+    'plus'
+];
+
+const contactPage = {
+    contactPageName: 'Contact',
+    
+    /* Simple fuctions for facilitate the live : START*/
+    
+    newElem(tagName) {
+        return document.createElement(tagName);
+    },
+    
+    setClass(target, className) {
```

Comment:
I like that decomposition with a lot of helpers.
I guess that methods should be moved to some other class or factory and make it like a helpers or so